### PR TITLE
Add tests considering optional attendees

### PR DESCRIPTION
**Pull request by KathleenQ**
Six extra tests are added considering optional attendees.
Please ignore format problems here. I will run clang to reformat this file in the next PR (i.e. PR #15 ) separately. (I don't want to mess up my changes and the initial necessary reformat.)

### CODE REVIEW

**Review by jupi3200 — COMMENTED** [review]
Please note that the common cycle of TDD is:

Write a test that fails -> make the code compile (but fail the test) -> fix the implementation to pass the test.

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 1]
```diff
@@ -121,6 +121,68 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredGivenBusyOptional() {
```
Comment (jupi3200) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java]: Could the test be named better?

**Comment by KathleenQ** [comment]
For the TDD process, yes, I realised that. So actually, the corresponding codes in FindMeetingQuery cannot pass these new added tests now. And, I will quickly fix FindMeetingQuery codes to pass all these tests as the next step after the tests are approved.

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java (hunk 1 above)
Comment (KathleenQ) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java]: Sure. I will change it to "onlyMandatoryAttendeesAreConsidered". I think it is clearer for this test.

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**